### ISSUE TITLE
qwen-vl-chat无法修改tokenization_qwen.py文件

### ISSUE THREAD

**Issue by SunreeenlonG1992** [xorbitsai/inference#2394]
### System Info / 系統信息

accelerate                               0.29.3
aiobotocore                              2.7.0
aiofiles                                 23.2.1
aiohttp                                  3.9.5
aioitertools                             0.11.0
aioprometheus                            23.12.0
aiosignal                                1.3.1
altair                                   5.3.0
annotated-types                          0.6.0
anyio                                    4.3.0
argcomplete                              3.5.0
async-timeout                            4.0.3
asyncer                                  0.0.7
attrdict                                 2.0.1
attrs                                    23.2.0
auto_gptq                                0.7.1
autoawq                                  0.2.5
autoawq_kernels                          0.0.6
av                                       13.0.0
bcrypt                                   4.1.3
bidict                                   0.23.1
bitsandbytes                             0.43.1
black                                    24.8.0
botocore                                 1.31.64
certifi                                  2024.2.2
cffi                                     1.16.0
chainlit                                 1.2.0
charset-normalizer                       3.3.2
chevron                                  0.14.0
click                                    8.1.7
cloudpickle                              3.0.0
cmake                                    3.30.3
colorama                                 0.4.6
coloredlogs                              15.0.1
contourpy                                1.2.1
cryptography                             42.0.8
cycler                                   0.12.1
dataclasses-json                         0.6.7
datamodel-code-generator                 0.26.1
datasets                                 2.19.0
Deprecated                               1.2.14
diffusers                                0.30.0
dill                                     0.3.8
distro                                   1.9.0
dnspython                                2.6.1
docstring_parser                         0.16
ecdsa                                    0.19.0
einops                                   0.8.0
email_validator                          2.2.0
eva-decord                               0.6.1
exceptiongroup                           1.2.1
fastapi                                  0.110.3
ffmpy                                    0.3.2
filelock                                 3.13.4
filetype                                 1.2.0
fire                                     0.6.0
fonttools                                4.51.0
frozenlist                               1.4.1
fsspec                                   2023.10.0
fvcore                                   0.1.5.post20221221
gekko                                    1.2.1
genson                                   1.3.0
googleapis-common-protos                 1.65.0
gradio                                   4.26.0
gradio_client                            0.15.1
grpcio                                   1.66.1
h11                                      0.14.0
httpcore                                 1.0.5
httpx                                    0.27.0
huggingface-hub                          0.24.5
humanfriendly                            10.0
idna                                     3.7
importlib_metadata                       8.2.0
importlib_resources                      6.4.0
inflect                                  5.6.2
iopath                                   0.1.10
isort                                    5.13.2
Jinja2                                   3.1.3
jj-pytorchvideo                          0.1.5
jmespath                                 1.0.1
joblib                                   1.4.2
jsonschema                               4.21.1
jsonschema-specifications                2023.12.1
kiwisolver                               1.4.5
Lazify                                   0.4.0
lit                                      18.1.8
literalai                                0.0.607
loguru                                   0.7.2
markdown-it-py                           3.0.0
MarkupSafe                               2.1.5
marshmallow                              3.22.0
matplotlib                               3.9.2
mdurl                                    0.1.2
modelscope                               1.16.0
mpmath                                   1.3.0
multidict                                6.0.5
multiprocess                             0.70.16
mypy-extensions                          1.0.0
nest-asyncio                             1.6.0
networkx                                 3.3
numpy                                    1.26.4
nvidia-cublas-cu11                       11.10.3.66
nvidia-cublas-cu12                       12.1.3.1
nvidia-cuda-cupti-cu11                   11.7.101
nvidia-cuda-cupti-cu12                   12.1.105
nvidia-cuda-nvrtc-cu11                   11.7.99
nvidia-cuda-nvrtc-cu12                   12.1.105
nvidia-cuda-runtime-cu11                 11.7.99
nvidia-cuda-runtime-cu12                 12.1.105
nvidia-cudnn-cu11                        8.5.0.96
nvidia-cudnn-cu12                        8.9.2.26
nvidia-cufft-cu11                        10.9.0.58
nvidia-cufft-cu12                        11.0.2.54
nvidia-curand-cu11                       10.2.10.91
nvidia-curand-cu12                       10.3.2.106
nvidia-cusolver-cu11                     11.4.0.1
nvidia-cusolver-cu12                     11.4.5.107
nvidia-cusparse-cu11                     11.7.4.91
nvidia-cusparse-cu12                     12.1.0.106
nvidia-ml-py                             12.560.30
nvidia-nccl-cu11                         2.14.3
nvidia-nccl-cu12                         2.18.1
nvidia-nvjitlink-cu12                    12.4.127
nvidia-nvtx-cu11                         11.7.91
nvidia-nvtx-cu12                         12.1.105
openai                                   1.35.13
opencv-contrib-python                    4.10.0.84
opentelemetry-api                        1.27.0
opentelemetry-exporter-otlp              1.27.0
opentelemetry-exporter-otlp-proto-common 1.27.0
opentelemetry-exporter-otlp-proto-grpc   1.27.0
opentelemetry-exporter-otlp-proto-http   1.27.0
opentelemetry-instrumentation            0.48b0
opentelemetry-proto                      1.27.0
opentelemetry-sdk                        1.27.0
opentelemetry-semantic-conventions       0.48b0
optimum                                  1.22.0
orjson                                   3.10.1
packaging                                23.2
pandas                                   2.2.2
parameterized                            0.9.0
passlib                                  1.7.4
pathspec                                 0.12.1
peft                                     0.10.0
Pillow                                   9.5.0
pip                                      23.3.1
platformdirs                             4.3.6
portalocker                              2.10.1
protobuf                                 4.25.5
psutil                                   5.9.8
pyarrow                                  16.0.0
pyarrow-hotfix                           0.6
pyasn1                                   0.6.0
pycparser                                2.22
pydantic                                 2.9.2
pydantic_core                            2.23.4
pydub                                    0.25.1
Pygments                                 2.17.2
PyJWT                                    2.9.0
pynvml                                   11.5.0
pyparsing                                3.1.2
pyre-extensions                          0.0.29
python-dateutil                          2.9.0.post0
python-dotenv                            1.0.1
python-engineio                          4.9.1
python-jose                              3.3.0
python-multipart                         0.0.9
python-socketio                          5.11.4
pytz                                     2024.1
PyYAML                                   6.0.1
quantile-python                          1.1
qwen-vl-utils                            0.0.8
referencing                              0.34.0
regex                                    2024.4.16
requests                                 2.31.0
rich                                     13.7.1
rouge                                    1.0.1
rpds-py                                  0.18.0
rsa                                      4.9
ruff                                     0.4.1
s3fs                                     2023.10.0
safetensors                              0.4.3
scikit-learn                             1.5.1
scipy                                    1.13.0
semantic-version                         2.10.0
sentence-transformers                    3.0.1
sentencepiece                            0.2.0
setuptools                               68.2.2
shellingham                              1.5.4
shtab                                    1.7.1
simple-websocket                         1.0.0
six                                      1.16.0
sniffio                                  1.3.1
sse-starlette                            2.1.0
starlette                                0.37.2
sympy                                    1.12
syncer                                   2.0.3
tabulate                                 0.9.0
tblib                                    3.0.0
termcolor                                2.4.0
threadpoolctl                            3.5.0
tiktoken                                 0.7.0
timm                                     1.0.7
tokenizers                               0.19.1
toml                                     0.10.2
tomli                                    2.0.1
tomlkit                                  0.12.0
toolz                                    0.12.1
torch                                    2.0.1
torchvision                              0.15.2
tqdm                                     4.66.2
transformers                             4.40.0
transformers-stream-generator            0.0.5
triton                                   2.0.0
trl                                      0.8.5
typer                                    0.11.1
typing_extensions                        4.11.0
typing-inspect                           0.9.0
tyro                                     0.8.3
tzdata                                   2024.1
uptrace                                  1.26.0
urllib3                                  2.0.7
uvicorn                                  0.25.0
uvloop                                   0.19.0
watchfiles                               0.20.0
websockets                               11.0.3
wheel                                    0.41.2
wrapt                                    1.16.0
wsproto                                  1.2.0
xformers                                 0.0.20
xinference                               0.15.3
xoscar                                   0.3.2
xxhash                                   3.4.1
yacs                                     0.1.8
yarl                                     1.9.4
zipp                                     3.20.0
zstandard                                0.22.0

### Running Xinference with Docker? / 是否使用 Docker 运行 Xinfernece？

- [ ] docker / docker
- [X] pip install / 通过 pip install 安装
- [ ] installation from source / 从源码安装

### Version info / 版本信息

xinference0.15.3

### The command used to start Xinference / 用以启动 xinference 的命令

xinference-local --host 0.0.0.0 --port 9997

### Reproduction / 复现过程

因为启动qwen-vl-chat-7b时缺少SimSun.ttf,所以我下载后尝试修改tokenization_qwen.py中的SimSun.ttf指定路径，当时每当我重启模型，okenization_qwen.py文件都会回到我修改之前
<img width="971" alt="image" src="https://github.com/user-attachments/assets/4b4a936b-7bbc-4bf7-b89a-7230aa0c6360">

> 

### Expected behavior / 期待表现

希望qwen-vl-chat-7b的tokenization_qwen.py支持修改

**Comment by qinxuye** [comment]
你可以自己 clone ，然后修改，通过 --model_path 或者界面中的 model_path 指定成你自己的路径。

**Comment by Shi-Hongkun** [comment]
我遇到了同样的问题
